### ISSUE TITLE
[Bug] Cameras receiving "out of range" only on multiplayer

### ISSUE THREAD

**Issue by griffinpuc** [northwesttrees-gaming/CCTV-Craft#30]
### Current Behavior

Cameras receieve out of range when vieing on multiplayer, while in my singleplayer world viewing a camera at the same distances and farther would work perfectly fine. Im not talking large ranges either, this is like 10 blocks away it stops working. Is there a configuration option I am missing somewhere? Please advise, thank you!

### Mod Build

unknown

### Minecraft Release

1.20.1

### Crash Report

None

**Comment by northwesttreesgaming** [comment]
Sounds like a server-side issue on CCTV Craft's end I will run a few tests and see if I can't fix this in 4.8.0's release.
Thanks for reporting the issue.

**Comment by northwesttreesgaming** [comment]
I just tested 4.7.0 on my test server, I could not reproduce the issue.
I tried a few things but it seems to be working as intended.

If it is possible, could you set up a basic forge server on your computer make a new world and then try to reproduce the issue?
If you can get it to do it again zip the world file and attach it to this issue. I will be able to do more with the world but as far as I can see right now it seems to be working fine. But it could be a harder bug to find than what I have been able to weed out of the code.

**Comment by griffinpuc** [comment]
@northwesttreesgaming Thanks for the reply, today I restarted the server and client and testing in the same world it seems to be working now. Im not sure if restarting the server made a difference (since the last time I tried it was the first time running the mod). Its a bit odd since last night I was really struggling to get the cameras to work, but today after the restart trying them in similar positions they seem to be working just fine.

Its also possible there is another mod conflict since I am running a few other mods in tandem with this.

Thanks for the reply! I will close this now since it seems to be resolved from the server restart.
